packages/vuetify/src/components/VDataTable/VDataTableHeader.sass

```diff
@@ -27,6 +27,8 @@
       pointer-events: auto
       cursor: pointer
       outline: 0
+      .v-data-table-header__icon
+        line-height: .9
```

Comment:
What was the determining factor of this value?

Reply:
Hi @johnleider ,

Thanks for the comment.
There is no specific reason for this value. 

However, by assigning line height to `0.9`, height of this dom will be decreased down to `20px` which ensures its size does not exceed header column bottom line.

This is only solution so far I can think of by touching neither `font-size` nor anything that affects the icon's eventual size.

Please advice any better solution.  

![2021_01_06_0kc_Kleki](https://user-images.githubusercontent.com/5698884/103716995-78241580-5018-11eb-9eb2-b902e439c8a1.png)

Reply:
Thoughts @nekosaur ?